packages/functional-tests/src/test-groups/functional-test-group.ts

```diff
@@ -0,0 +1,120 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+import { GuidGenerator } from 'common';
+import { Logger, LogLevel } from 'logger';
+import { OnDemandPageScanRunResultProvider, WebApiErrorCode } from 'service-library';
+import { isNullOrUndefined } from 'util';
+import { A11yServiceClient } from 'web-api-client';
+
+import { SerializableResponse, TestContextData } from '../test-group-data';
+
+export abstract class FunctionalTestGroup {
```

Comment:
Can we have option to run test in arbitrary environment only?

Reply:
Updated.